modules/disabling-redirect-private-storage-endpoint-azure.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="disabling-redirect-private-storage-endpoint-azure"]
+= Optional: Disabling redirect when using a private storage endpoint on Azure
+
+By default, redirect is enabled when using the image registry. Redirect allows off-loading of traffic from the registry pods into the registry, which makes pull faster. When redirect is enabled, users from outside of the cluster are unable to pull images from the registry. 
+
+In some cases, users might want to disable redirect so that users from outside of the cluster can pull images from the registry. 
+
+Use the following procedure to disable redirect.
+
+.Prerequisites
+
+* You have configured the image registry to run on Azure
+* You have configured a route.
+
+.Procedure
+
+* Enter the following command to disable redirect on the image 
+registry configuration:
++
+[source,terminal]
+----
+$ oc patch configs.imageregistry cluster --type=merge -p '{"spec":{"disableRedirect": true}}'
+----
+
+.Verification
++
+[NOTE]
+====
+The following procedure requires proper verification from a user outside of the cluster.
```

Comment:
This sentence is correct, but the steps below are for verifying the procedure from within the cluster.
Remove the `oc debug node/<node-name>` and `chroot /host` steps and use the external registry route.